mathics/builtin/control.py

```diff
@@ -599,8 +599,36 @@ def apply(self, evaluation):
 
         raise AbortInterrupt
 
-# class Return(Builtin):
-#    pass
+
+class Return(Builtin):
+    '''
+    <dl>
+    <dt>'Return[$expr$]'
+      <dd>aborts a function call and returns $expr$.
+    </dl>
+
+    >> f[x_] := (If[x < 0, Return[0]]; x)
+    >> f[-1]
+     = 0
+
+    >> g[x_] := (Do[If[x < 0, Return[0]], {2, 1, 0, -1}]; x)
```

Comment:
It's an example I copied off the MMA `Return` docs. I don't think it's a good example. I'll fix it :)